bitnami/elasticsearch/templates/_helpers.tpl

```diff
@@ -403,54 +423,60 @@ Add environment variables to configure database values
 {{- end }}
 {{- if and .Values.security.tls.usePemCerts (or .Values.security.tls.keyPassword .Values.security.tls.passwordsSecret) }}
 - name: ELASTICSEARCH_KEY_PASSWORD
-  value: {{ .Values.security.tls.keyPassword | quote }}
+  valueFrom:
+    secretKeyRef:
+      name: {{ include "elasticsearch.tlsPasswordsSecret" . }}
+      key: key-password
```

Comment:
Should this key come from another helper? I mean, make it customizable so you don't force users to use a specific key if they're setting `.Values.security.tls.passwordsSecret`

Reply:
Agree, it would be a nice feature, especially for users using Cert-Manager. I will implement it.